packages/common/core-interfaces/src/events/listeners.ts

```diff
@@ -5,7 +5,10 @@
 
 /**
  * `true` iff the given type is an acceptable shape for a {@link Listeners | event} listener
- * @public
+ * @system @public
+ *
+ * @remarks
+ * Deprecated in `@fluidframework/tree`. Consider importing from `@fluidframework/core-interfaces` instead.
```

Comment:
Should we add the "from `fluid-framework` or " text in these comments too?

Reply:
Deprecation via tree is working but it shows as deprecated while importing from fluid-framework too.

Reply:
I built the branch. I think you must have something stale. In the current version, fluid-framework doesn't export these APIs - they aren't in fluid-framework/src/index.ts as re-exports from core-interfaces.